## 71% of end-end tests have passed
```
Total executed tests: 14
Failed tests: 2
Expected to fail tests: 2
Passed tests: 10
```
```
IDs of failed tests: 703133,702843 
```
```
IDs of expected to fail tests: 741840,741841 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/android-wc-explorer&PR_ID=22348&APK_NAME=22348.apk&TR_CASE_IDS=703133,702843">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818723">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:246: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: zQ3...dWXh5 was not restored as a contact from waku backup!
E    Device 1: zQ3...Vacac was not restored as a contact from waku backup!
E    Device 1: admin_open was not restored from waku-backup!!
E    Device 1: member_open was not restored from waku-backup!!
E    Device 1: admin_closed was not restored from waku-backup!!
E    Device 1: member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESvv01nfvgMwSeI1GP">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818722">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` </div><div>Device 2: Wait for element `Button` for max 120s and click when it is available </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:349: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:336: in join_community
    self.join_button.wait_and_click(120)
../views/base_element.py:99: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by accessibility id:`show-request-to-join-screen-button` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESksDYnFdbRlnorTrm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkwF3EVfIuOFhMoNj">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818732">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/wallet/test_collectibles.py:146: in test_wallet_send_collectible
    self.wallet_view.slide_button_track.slide()
../views/base_view.py:107: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:288: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:274: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Collectibles list is not loaded sometimes]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818733">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `collectibles-tab` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/wallet/test_collectibles.py:158: in test_wallet_collectible_send_from_expanded_info_view
    self.wallet_view.get_collectible_element('Glitch Punks').wait_for_element().click()
../views/base_element.py:120: in wait_for_element
    raise TimeoutException(
 Device `1`: `CollectibleItemElement` by` xpath`: `//*[@content-desc='collectible-list-item']//*[contains(@text,'Glitch Punks')]/../..` is not found on the screen after wait_for_element 
```

<b>[[Collectibles list is not loaded sometimes]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (10)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818721">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXz9C680HLzXF0Quj">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYp9j8AGg10er1FFf">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818720">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESvv01nfvgMwSeI1GP">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818731">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818725">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818724">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818729">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818726">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818727">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818730">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3818728">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/android/app/src/main/AndroidManifest.xml
+++ b/android/app/src/main/AndroidManifest.xml
@@ -102,6 +102,7 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data android:scheme="status-app" />
                 <data android:scheme="ethereum" />
+                <data android:scheme="wc" />
             </intent-filter>
         </activity>
         <activity android:name="com.facebook.react.devsupport.DevSettingsActivity"/>

--- a/src/status_im/common/router.cljs
+++ b/src/status_im/common/router.cljs
@@ -347,6 +347,10 @@
       (string/starts-with? uri constants/local-pairing-connection-string-identifier)
       (cb {:type :localpairing :data uri})
 
+      (string/starts-with? uri constants/wc-connection-string-identifier)
+      (cb {:type :wallet-connect
+           :uri  uri})
+
       :else
       (cb {:type :undefined
            :data uri}))))

--- a/src/status_im/constants.cljs
+++ b/src/status_im/constants.cljs
@@ -203,7 +203,7 @@
          (str "wallet/" regx-string-any-ascii "+")])
        ")$"))
 (def regx-universal-link (re-pattern regx-string-universal-link))
-(def regx-deep-link #"((^ethereum:.*)|(^status-app://[\x00-\x7F]+$))")
+(def regx-deep-link #"((^ethereum:.*)|(^status-app://[\x00-\x7F]+)|(^wc://.*))")
 (def regx-ens #"^(?=.{5,255}$)([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.[a-zA-Z]{2,}$")
 (def regx-starts-with-uuid #"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
 

--- a/src/status_im/constants.cljs
+++ b/src/status_im/constants.cljs
@@ -529,3 +529,5 @@
 (def ^:const go-waku-url "https://github.com/waku-org/go-waku")
 (def ^:const status-go-url "https://github.com/status-im/status-go")
 (def ^:const status-mobile-url "https://github.com/status-im/status-mobile")
+
+(def ^:const wc-connection-string-identifier "wc")

